recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "gensim" %}
+{% set version = "2.2.0" %}
+{% set sha256 = "eb099de1e50447c42e168a1a99de4721923688afc71b12fe522f79687a4fbb13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.11.3
+    - cython
+    - scipy >=0.18.1
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+  run:
+    - python
+    - numpy >=1.11.3
+    - scipy >=0.18.1
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+test:
+  requires:
+    - nose
+    - testfixtures
+    - morfessor
+    - scikit-learn
+    - pyemd
+    - python-annoy
+    - tensorflow    
+    - keras
+    
+  commands:
+    - nosetests --exe -v gensim
+    
+about:
+  home: http://github.com/RaRe-Technologies/gensim
+  license: LGPL 3.0
+  license_file: COPYING
+  license_family: LGPL
+  summary: 'A library for topic modelling and document indexing'
+  description: |
```

Comment:
Please use the official tagline and description (see https://github.com/RaRe-Technologies/gensim).